modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+  * You now see a form that guides you through the process of creating a RoleBinding:
+. For *Binding Type*, select *Namespace Role Binding (RoleBinding)*.
+  * If you only select *All Projects*, then you only need to provide the *Binding Type*.
+  * If the project is already selected as the one you want to create the RoleBinding for, the project does not ask for *Binding Type* because it assumes it is a RoleBinding.
+. Under *Role Binding*, provide a RoleBinding *Name* and *Namespace*.
+  * For the *Namespace*, specify the one you are granting permissions for.
+. For *Role*, select the *Cluster Role admin (CR admin)* from the dropdown list.
+. Under *Subject*, select *Service Account* and provide the *Subject Namespace* and *Name*. For example:
+  * Subject Namespace: openshift-gitops
+  * Subject Name: argocd-cluster-argocd-application-controller
+. To create the RoleBinding, select *Create*.
+
+.Configuring clusters
+
+After you deploy an ArgoCD instance, you can manage your cluster configurations:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Update the existing *Subscription Object* for the GitOps operator.
+. Go the spec and add, *ARGOCD_CLUSTER_CONFIG_NAMESPACES*.
+. On the OpenShift Console, go to *Administration* > *CustomResourceDefinition* > *Subscription* > *Instances* then select *openshift-gitops-operator*.
+. Select the *YAML* tab, edit the Subscription yaml, and add:
+
+```
+spec:
+  config:
+    env:
+    - name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
+      Value: openshift-gitops, <namespace where argocd instance is installed>
+```
+
+[id="setting-up-an-additional-permissions-for-argocd_{context}"]
+== Setting up additional permissions for ArgoCD
+
+If you want to expand the permissions granted to ArgoCD, create Cluster Roles with additional permissions and then a new ClusterRoleBinding to associate them to a Service Account.
+
+For an example, extend the permissions for the pre-installed ArgoCD instance to list the secrets for all namespaces. To extend these permissions, you must create a cluster role:
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Navigate to *User Management* > *Roles* > *Create Role*.
+. Use the following ClusterRole Yaml template to add rules to specify the additional permissions, then select *Create*:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: secrets-cluster-role
+rules:
+- apiGroups: [""] #specifies core api groups
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list"]
+```
+
+.Creating ClusterRoleBinding
+
+. Go to the table of contents and make sure you are in the *Administrator* perspective and not the *Developer* perspective.
+. Navigate to *User Management* > *Role Bindings* > *Create Binding*.
```

Comment:
In the *Administrator* perspective of the web Console, navigate to... 
Arrows should use this format -> and not > in all instances in this PR